components/script/dom/webidls/WebGLRenderingContext.webidl

```diff
@@ -489,14 +489,15 @@ interface WebGLRenderingContextBase
                            GLenum srcAlpha, GLenum dstAlpha);
 
     // typedef (ArrayBuffer or ArrayBufferView) BufferDataSource;
-    //void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
     // FIXME(dmarcos) The function below is the original function in the webIdl:
     // void bufferData(GLenum target, BufferDataSource? data, GLenum usage);
     // The Code generator doesn't handle BufferDataSource so we're using 'object?'
     // in the meantime, and marking the function as [Throws], so we can handle
     // the type error from inside.
     [Throws]
     void bufferData(GLenum target, object? data, GLenum usage);
+    [Throws]
+    void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
```

Comment:
This needs to be `[Throws]` just because the overload is `[Throws]` too, right?

If so, mind adding a `FIXME` here so when we have proper `ArrayBuffer` WebIDL support we remove both?